docs/graphql/manual/auth/authentication/unauthenticated-access.rst

```diff
@@ -22,7 +22,7 @@ Once you have configured authentication, by default Hasura GraphQL engine will r
 receives. 
```

Comment:
`Once you have configured authentication` -> Once you have configured an admin-secret [link to securing your endpoint page]